Tests/AK/TestOptional.cpp

```diff
@@ -247,7 +247,6 @@ TEST_CASE(move_optional_reference)
     y = move(x);
     EXPECT_EQ(y.has_value(), true);
     EXPECT_EQ(y.value(), 3);
-    EXPECT_EQ(x.has_value(), false);
```

Comment:
Is this intended?

Reply:
Yes. Although we could probably keep the move constructor non-trivial to keep this behavior.

Reply:
Not really an issue, but please mention (the reason for) its removal in the commit description.